docs/content/preview/drivers-orms/java/yugabyte-jdbc.md

```diff
@@ -128,13 +128,13 @@ The following table describes the connection parameters required to connect, inc
 | database | Database name | yugabyte |
 | user | User connecting to the database | yugabyte |
 | password | User password | yugabyte |
-| `load-balance` | [Uniform load balancing](../../smart-drivers/#cluster-aware-connection-load-balancing) | Defaults to upstream driver behavior unless set to one of the allowed values other than 'false' |
```

Comment:
at line 102 add "Note that v4 of HikariCP is required because the YugabyteDB JDBC Driver requires Java 8."